testsuite/features/reposync/allcli_update_activationkeys.feature

```diff
@@ -127,6 +127,13 @@ Feature: Update activation keys
     And I check "SUSE-Manager-Proxy-5.0-Updates for x86_64"
     And I click on "Update Activation Key"
     Then I should see a "Activation key Proxy Key x86_64 has been modified" text
+    # TODO: remove when we use combustion
+    # 'expect' is needed for Cobbler and Retail tests
```

Comment:
As temporary solution, lgtm.
But please, could you already prepare a PR to include this package in our [combustion config](https://github.com/uyuni-project/sumaform/blob/master/backend_modules/libvirt/host/combustion)?

Reply:
I can, but how would i test it works?

BTW, I did not even test this code yet. It could be it does not work (missing repos, etc.). If that's the case i need to create a first version for cloud-init ::cry: .